src/main/java/Main.java

```diff
@@ -0,0 +1,172 @@
+import spark.Spark;
+import spark.ModelAndView;
+import spark.Session;
+import spark.template.mustache.MustacheTemplateEngine;
+
+import java.util.*;
+
+/**
+ * Created by vasantia on 7/13/16.
+ */
+public class Main {
+
+    public static final String SESSION_USER = "userName";
+    static Map<String, User> users = new HashMap<>();
+    static List<Bottle> allBottles = new ArrayList<>();
+
+    public static void main(String[] args) {
+
+        Spark.staticFileLocation("public");
+        Spark.init();
+
+        Spark.get("/", (request, response) -> {
+            Map m = new HashMap();
+            Session session = request.session();
+            String name = session.attribute(SESSION_USER);
+            User user = users.get(name);
+
+//            List<Bottle> bottlePages = new ArrayList<>();
+//            String pageString = request.queryParams("page");
+//            int page = Integer.valueOf(pageString);
+//            int itemsPerPage = 5;
+//            int newPageStart = page * itemsPerPage;
+//            String url = "/";
+//
+//            for (Bottle bottle : user.myBottles){
+//                bottlePages.add(bottle);
+//                }
+//
+//            }
+//            for (int i = newPageStart; i < (newPage + itemsPerPage); i++){
+//
+//                url += "?page=" + page;
+//            }
+
+            if(user == null){
+                m.put("allBottles", allBottles);
+                return new ModelAndView(m, "index.html");
+            }
+            else {
+                m.put("user", user);
+                m.put("allBottles", allBottles);
+                return new ModelAndView(m, "index.html");
+            }
+            //m.put("bottlePages", bottlePages);
+//            m.put("url", url);
+//            m.put("itemsPerPage", itemsPerPage);
+//            m.put("page", page);
+//            m.put("newPage", newPage);
+        }, new MustacheTemplateEngine());
+
+        Spark.get("/login", (request, response) -> {
```

Comment:
This method really isn't needed.  It could be combined with the "/" get route as it is the same thing.